dockers/docker-fpm-frr/base_image_files/platform_utils

```diff
@@ -16,3 +16,40 @@ fi
 
 # Export SMARTSWITCH for other scripts
 export SMARTSWITCH
+
+IS_SUPERVISOR=false
+PLATFORM_ENV_CONF_FILE=/usr/share/sonic/device/$PLATFORM/platform_env.conf
+if [ -f "$PLATFORM_ENV_CONF_FILE" ]; then
+    source $PLATFORM_ENV_CONF_FILE
+    if [ -v supervisor ]; then
+        if [ $supervisor -eq 1 ]; then
+            IS_SUPERVISOR=true
+        else
+            IS_SUPERVISOR=false
+        fi
+    else 
+        IS_SUPERVISOR=false
+    fi
+fi
+
+# Export IS_SUPERVISOR for other scripts
+export IS_SUPERVISOR
+
+# Export IS_CHASSIS_MODULE for other scripts
+IS_CHASSIS_MODULE=false
+PLATFORM_CHASSISDB_CONF_FILE=/usr/share/sonic/device/$PLATFORM/chassisdb.conf
+ETC_SONIC_CHASSISDB_CONF_FILE=/etc/sonic/chassisdb.conf
```

Comment:
Could you please add some comments here to explain the logic? Please clarify the files/flags on chassis, vs pizza box

Reply:
> Could you please add some comments here to explain the logic? Please clarify the files/flags on chassis, vs pizza box

Added comments as suggestion

Reply:
@mlok-nokia - would it be possible to add flags in platform_env.conf file to identify chassis, line card and disagg chassis to make this logic cleaner?

Reply:
> @mlok-nokia - would it be possible to add flags in platform_env.conf file to identify chassis, line card and disagg chassis to make this logic cleaner?
The current design/implementation does not add any new flag. The reason is avoid touch other vendor's platfrom_env file.  And also it needs to handle the backward compatibility for platform  x86_64-nvidia_sn4280-r0  which has /etc/sonic/chassisdb.conf but No platform_env.conf file